core/store/src/trie/prefetching_trie_storage.rs

```diff
@@ -334,7 +337,7 @@ impl PrefetchStagingArea {
                 Some(_) => (),
                 None => return None,
             }
-            std::thread::sleep(std::time::Duration::from_micros(1));
```

Comment:
unreleated to the PR, but this one here looks quite suspect. Should this have a Condvar instead of a busy-wait here?

Reply:
https://github.com/near/nearcore/issues/7723